features/did_you_know.feature

```diff
@@ -0,0 +1,8 @@
+Feature: Some commands print hints about related commands and options
+
+  Scenario: A hint is displayed after running the rake command
+    Given a file named "bin/rake" with "binstub"
+    And a file named "tmp/global_settings.yml" with "did_you_know: 100"
```

Comment:
Now that I see this in practice, let's rename the config option once more: `hint_probability`.